float64 height
float64 width

float64 fow

float64 angle_roll
float64 angle_pitch
float64 angle_yaw